lib/app.ts

```diff
@@ -0,0 +1,102 @@
+/*
+Copyright 2016 Resin.io
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Import most things with types if possible.
+import Opts = require('node-getopt');
```

Comment:
Ah, makes sense. I think maybe I'd parsed this as `var Opts = require('node-getopt')`, which is the case we need to avoid. `import X = ...` I'm fine with, if that's what works.